source/adios2/core/VariableBase.h

```diff
@@ -145,18 +145,40 @@ class VariableBase
         ConvertUint64VectorToSizetVector(sel.m_Start, m_MemoryOffsets);
     }
 
+    /**
+     * Set the steps for the variable to read from. The pointer passed at
+     * reading must be able to hold enough memory to store multiple steps in a
+     * single read.
+     * @param fromStep  The first step to read. Steps start from 0
+     * @param nSteps    Number of consecutive steps to read at once.
+     *
+     */
+    void SetStepSelection(const unsigned int fromStep,
+                          const unsigned int nSteps)
+    {
+        m_ReadFromStep = fromStep;
+        m_ReadNSteps = nSteps;
+    }
+
     /** Return the number of steps available for the variable
      *  @return Number of steps
      */
-    unsigned int GetNSteps() { return m_nsteps; }
+    unsigned int GetNSteps() { return m_NStepsAvailable; }
 
     ///< Should only be called by read engines
```

Comment:
doxygen ///<  documents text before  the message. 
Use /**   documentationa */ for the line below